code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -387,10 +387,15 @@
 		ticker.mode.thralls[ref] = list(H.mind)
 	else
 		ticker.mode.thralls[ref] += H.mind
+	var/datum/objective/protect/new_objective = new /datum/objective/protect
+	new_objective.owner = H.mind
+	new_objective:target = src.mind
+	new_objective.explanation_text = "You have been Enthralled by [src.real_name], the vampire. Follow their every command."
+	H.mind.objectives += new_objective
 	ticker.mode.enthralled.Add(H.mind)
 	ticker.mode.enthralled[H.mind] = src.mind
 	H.mind.special_role = "VampThrall"
-	H << "<span class='sinister'>You have been Enthralled by [name]. Follow their every command.</span>"
+	H << "<span class='sinister'>You have been Enthralled by [src.real_name]. Follow their every command.</span>"
```

Comment:
I had figured that name would be an issue if the vamp were disguised while biting a neck, then realized that that isn't possible since they can't wear a mask.